notebooks/04_model_select_and_optimize/azureml_hyperdrive_wide_and_deep.ipynb

```diff
@@ -7,7 +7,7 @@
     "<i>Copyright (c) Microsoft Corporation. All rights reserved.<br>\n",
```

Comment:
<p>any thoughts on why these values dropped so much?</p><p><br></p><p>also, i think using getattr is a bit obscure, it would be clearer to just call the metrics code directly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/847/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>